frontend/src/components/common/SideBar/index.tsx

```diff
@@ -36,14 +36,14 @@ export interface Props {
 
 export default function SideBar({
   children: content,
+  className,
   label,
   isOpen = false,
   onToggle,
   width = FILTERS_PANEL_EXPANDED_WIDTH_PX,
   position = Position.LEFT,
   SideBarWrapperComponent = SideBarWrapper,
   SideBarPositionerComponent = SideBarPositioner,
-  SideBarOpenButtonWrapperComponent = SideBarOpenButtonWrapper,
```

Comment:
The prop `SideBarOpenButtonWrapperComponent` is no longer in use. Added `className` to `Sidebar` to facilitate styling of the component (see https://emotion.sh/docs/styled#styling-any-component).